note.ts

```diff
@@ -0,0 +1,11 @@
+export interface Note {
+	type: "success" | "warning" | "danger" | "primary"
```

Comment:
We should probably not name them after the colors. Maybe `primary` should be called `default` or something.